shell/platform/darwin/ios/framework/Source/FlutterAppDelegate.mm

```diff
@@ -31,13 +31,6 @@ - (instancetype)init {
   return self;
 }
 
-- (void)dealloc {
```

Comment:
Don't forget to add `FLUTTER_ASSERT_ARC` under the imports when you migrate an implementation file.

Reply:
Done.